collabcloud/project-collabcloud
# Col-12-ProjectSettings

New Features:
- Ability to view a Project that you created
- Ability to update a Project that you created
- Ability to delete a Project that you created

Testing:
1. Fill in all the form fields and add a project in http://localhost:3000/projects/create
- [OPTIONAL] Note that if you want to pre-fill one of your repos from GitHub, you must set ```githubUsername = <your_username>``` on line 57 in CreateProjects.js
2. Open up PSQL terminal and do ```SELECT * FROM projects;``` to see your new project
3. Copy the pid from PSQL and paste it into line 22 on Project.js
4. Go to http://localhost:3000/user/project
- The project you added should now show up here. If it is not here, then you won't be able to test (let me know)
![image](https://user-images.githubusercontent.com/19757152/76505995-c8464180-6420-11ea-8e21-0d3036bdff93.png)

5. Play around with the "Settings", "Update", and "Delete" button (I suggest clicking on Delete last)
![image](https://user-images.githubusercontent.com/19757152/76506061-ea3fc400-6420-11ea-9299-fad33b7a539f.png)
